OpenEdXMobile/src/androidTest/java/org/edx/mobile/test/screenshot/test/DiscoveryLaunchScreenshotTests.java

```diff
@@ -9,11 +9,16 @@
 
     @Test
     public void testScreenshot_withCourseDiscoveryDisabled() {
```

Comment:
there should be another test case scenario when CourseDiscoveryDisabled and ProgramDiscoveryEnabled

Reply:
This scenario can't be exist because programs discovery can only be enable if the courses discovery is enabled